Products/zms/_objattrs.py

```diff
@@ -611,8 +611,11 @@ def attr(self, *args, **kwargs):
     def evalMetaobjAttr(self, *args, **kwargs):
       root = self
       request = self.REQUEST
-      id = request.get('ZMS_INSERT', self.meta_id)
       key = args[0]
+      if key=='standard_html' and request.get('ZMS_ZCATALOG_REINDEXING', False):

```

Comment:
This seems to be a very special case and should not be implemented in the general objattrs module. Otherwise there would be an implicit dependency to the catalog-adapter.